authorization/authorization.go

```diff
@@ -20,20 +20,23 @@ const (
 	// ResourceTypeSpace defines the string constant for the space resource type
 	ResourceTypeSpace = "openshift.io/resource/space"
 
-	// AdminRole is the constant used to denote the name of a resource's administrator role
-	AdminRole = "admin"
+	// adminRole is the internal constant used to denote the administrator role for various resources
+	adminRole = "admin"
+
+	// OrganizationAdminRole is the constant used to denote the name of the organization resource's administrator role
+	OrganizationAdminRole = adminRole
+
+	// SpaceAdminRole is the constant used to denote the name of a space resource's administrator role
+	SpaceAdminRole = adminRole
 
 	// SpaceContributorRole is the constant used to denote the name of the space's contributor role
 	SpaceContributorRole = "contributor"
 
 	// SpaceViewerRole is the constant used to denote the name of the space's viewer role
 	SpaceViewerRole = "viewer"
 
-	// OwnerRole is the constant used to denote the name of the organization, team or security group owner role
-	OwnerRole = "owner"
-
 	// ManageMembersScope is the scope required for users wishing to invite/remove other users to an organization, team or security group
-	ManageMembersScope = "manage_members"
+	ManageMembersScope = "manage"
```

Comment:
WDYT about:
manageScope = "manage"
ManageTeamMembersScope = manageScope

And replace manageSpaceScope by manageScope?

Reply:
good idea, updated